frontend/src/metabase/nav/components/SearchResults.tsx

```diff
@@ -34,7 +34,8 @@ export const SearchResults = ({
 }: SearchResultsProps) => {
   const dispatch = useDispatch();
 
-  const [debouncedSearchText, setDebouncedSearchText] = useState(searchText);
+  const [debouncedSearchText, setDebouncedSearchText] = useState<string>();
```

Comment:
Before, when `SearchResults` was rendered, it was calling the `useSearchListQuery` on the initial string and causing that extra call. Now, we just let the `useDebounce` hook trigger when the user types something, which will then force the `query` to only change (and cause the `useSearchListQuery` hook) to trigger only when